### PR TITLE
Updating UKIT 008

### PR DESCRIPTION
**Pull request by Mick3Lozzo**
Updating UKIT 008 instructions with the inclusion of RealSense D405 and its related cable.

### CODE REVIEW

File: docs/upgrade_kits/realsense_holder/support.md [hunk 1]
```diff
@@ -3,9 +3,14 @@
...
 
-This upgrade kit is meant to improve iCub's 3D perception and grasping capabilities by adding a RealSense camera plugged on the head, in addition to the other eye cameras. 
+This upgrade kit is meant to improve iCub's 3D perception and grasping capabilities by adding a RealSense camera plugged on the head, in addition to the other eye cameras.
 
-<center> ![](img/8.PNG) </center>
+It ensures the compatibility with more RealSense models.
+
+|       |       |      |
+| :---: | :---: | :---: |
+| <img src="img\11.PNG" height=250 > | <img src="img\8.PNG" height=250 > | <img src="img\9.PNG" height=250 > |
```
Comment (pattacini) [inline docs/upgrade_kits/realsense_holder/support.md:12]: We need forward slashes here.

File: docs/upgrade_kits/realsense_holder/support.md [hunk 2]
```diff
@@ -47,45 +53,62 @@ This upgrade kit is meant to improve iCub's 3D perception and grasping capabilit
...
+ <img src="img/6.PNG" height=400 alignement=center >
 
 **Assembly sequence**
 
 *Perform the assembly sequence as shown in the following images.*
 
 - First step, remove the two screws V4-12--_-_U5933_C from the side of the face cover as can be seen in the image. Set them aside, just in case you want to successively unmount the RealSense holder.
 
-<center> ![](img/2.PNG) </center>
+ <img src="img/2.PNG" height=400 alignement=center >
```
Comment (pattacini) [inline docs/upgrade_kits/realsense_holder/support.md:100]: Did you check that the alignment specified this way actually works?

File: docs/upgrade_kits/realsense_holder/support.md [hunk 3]
```diff
@@ -47,45 +53,62 @@ This upgrade kit is meant to improve iCub's 3D perception and grasping capabilit
...
+ <img src="img/3.PNG" height=400 alignement=center >
 
 - Third step, fix the chosen RealSense model to the holder usign the screws V3-8--_-_U5933_C. At this point is possible to plug the USB Type-C cable cod. 15365 and use the cable ties cod. 531 to fix it around the head.
 
-<center> ![](img/4.PNG) </center>
-<center> ![](img/5.PNG) </center>
+ <img src="img/4.PNG" height=400 alignement=center >
+ <img src="img/5.PNG" height=400 alignement=center >
+
+ **NOTE:** As mentioned above, the RealSense D405 uses a different cable too.
```
Comment (pattacini) [inline docs/upgrade_kits/realsense_holder/support.md:111]: Same comment as above.

**Review by pattacini — CHANGES_REQUESTED** [review]

File: docs/upgrade_kits/realsense_holder/support.md (hunk 1 above)
Comment (Mick3Lozzo) [reply docs/upgrade_kits/realsense_holder/support.md:12]: Corrected @pattacini

File: docs/upgrade_kits/realsense_holder/support.md [hunk 4]
```diff
@@ -42,8 +42,9 @@ It ensures the compatibility with more RealSense models.
 |   :---: |    :-----------: |:---:|     :------------: |   :---:   |
 |    1   |	INTEL_REALSENSE_D435i  |  |Intel Realsense D435i with IMU, range from 0.2 to 10m, FOV 90 de | 14567 |
 
-!!! note "note"
-    [Logic schematics](https://github.com/icub-tech-iit/electronics-wiring-public/tree/master/icub-upgrade-kits/kit_008/15389%20-%20iCub2.5%20Head%20RealSense%20upgrade%20kit%20(electronic%20parts)) 
+> **Note 1**
+> 
+> [Logic schematics](https://github.com/icub-tech-iit/electronics-wiring-public/tree/master/icub-upgrade-kits/kit_008/15389%20-%20iCub2.5%20Head%20RealSense%20upgrade%20kit%20(electronic%20parts))
```
Comment (pattacini) [inline docs/upgrade_kits/realsense_holder/support.md:47]: Wrong

File: docs/upgrade_kits/realsense_holder/support.md [hunk 5]
```diff
@@ -53,11 +54,13 @@ It ensures the compatibility with more RealSense models.
...
+> 
+> See also [historical review](https://github.com/Mick3Lozzo/documentation/blob/master/docs/upgrade_kits/realsense_holder/support.md#historical-review) for further details. 
 
-**NOTE** : see also [historical review](https://github.com/Mick3Lozzo/documentation/blob/master/docs/upgrade_kits/realsense_holder/support.md#historical-review) for further details. 
-
-!!! note "note"
-    [Harness schematics](https://github.com/icub-tech-iit/electronics-wiring-public/tree/master/icub-upgrade-kits/kit_008/15393%20-%20iCub2.5%20Head%20RealSense%20upgrade%20kit%20(wiring%20parts)) 
+> **Note 3**
+> 
+> See [Harness schematics](https://github.com/icub-tech-iit/electronics-wiring-public/tree/master/icub-upgrade-kits/kit_008/15393%20-%20iCub2.5%20Head%20RealSense%20upgrade%20kit%20(wiring%20parts))
```
Comment (pattacini) [inline docs/upgrade_kits/realsense_holder/support.md]: Wrong

File: docs/upgrade_kits/realsense_holder/support.md [hunk 6]
```diff
@@ -87,9 +91,11 @@ The holder has been designed to be compatible with the following RealSense model
...
 The RealSense holder can be mounted on the top of iCub head by replacing two side screws fixing the face cover, and in the kit are included and USB Type-C cable, screws and cable ties. The kit does NOT include the RealSense. It is recommended to replace the short cable included in the RealSense retail package with the longer one provided with the upgrade kit.
 
-**NOTE**: The following instructions are provided for assembling the D435i model of RealSense camera.
-
- <img src="img/6.PNG" height=400 alignement=center >
+> **Note 4**
+>
+>   The following instructions are provided for assembling the D435i model of RealSense camera.
+> 
+> <img src="img/6.PNG" height=400 alignement=center >
```
Comment (pattacini) [inline docs/upgrade_kits/realsense_holder/support.md:98]: Wrong

File: docs/upgrade_kits/realsense_holder/support.md (hunk 2 above)
Comment (Mick3Lozzo) [reply docs/upgrade_kits/realsense_holder/support.md]: I tried something other but didn't work. For now, I'm not centring the pictures anymore.

File: docs/upgrade_kits/realsense_holder/support.md [hunk 7]
```diff
@@ -47,45 +53,62 @@ This upgrade kit is meant to improve iCub's 3D perception and grasping capabilit
...
+|	INTEL_REALSENSE_D405   |USB Shielded I/O Cable Assembly, USB 3.2 Gen 1, Male A to Male Micro-B, 3.0m length, black | 17308 |
+
 
 
 ## Assembly instruction
 
 The RealSense holder can be mounted on the top of iCub head by replacing two side screws fixing the face cover, and in the kit are included and USB Type-C cable, screws and cable ties. The kit does NOT include the RealSense. It is recommended to replace the short cable included in the RealSense retail package with the longer one provided with the upgrade kit.
 
-<center> ![](img/6.PNG) </center>
+**NOTE**: The following instructions are provided for assembling the D435i model of RealSense camera.
```
[the remark this answers is not in the document]
Comment (Mick3Lozzo) [reply docs/upgrade_kits/realsense_holder/support.md]: The title is out of the admonition box. What I am doing wrong @pattacini? I'm following this format:  
  
!!! note "Phasellus posuere in sem ut cursus" Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla et euismod nulla. Curabitur feugiat, tortor non consequat finibus, justo purus auctor massa, nec semper lorem quam in massa.

File: docs/upgrade_kits/realsense_holder/support.md (hunk 2 above)
Comment (pattacini) [reply docs/upgrade_kits/realsense_holder/support.md]: We need to center images. So, please stick to the previous format that was relying on `<center>...</center>`.

File: docs/upgrade_kits/realsense_holder/support.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (pattacini) [reply docs/upgrade_kits/realsense_holder/support.md]: Before it wasn't correct. I've seen you fixed it in the latest commits.

File: docs/upgrade_kits/realsense_holder/support.md (hunk 2 above)
Comment (Mick3Lozzo) [reply docs/upgrade_kits/realsense_holder/support.md]: Re-centered pictures @pattacini

File: docs/upgrade_kits/realsense_holder/support.md [hunk 8]
```diff
@@ -47,45 +54,71 @@ This upgrade kit is meant to improve iCub's 3D perception and grasping capabilit
...
 
-<center> ![](img/3.PNG) </center>
+<center> <img src="img/3.PNG" height=400> </center>
 
 - Third step, fix the chosen RealSense model to the holder usign the screws V3-8--_-_U5933_C. At this point is possible to plug the USB Type-C cable cod. 15365 and use the cable ties cod. 531 to fix it around the head.
 
-<center> ![](img/4.PNG) </center>
-<center> ![](img/5.PNG) </center>
+<center> <img src="img/4.PNG" height=400> </center>
+<img src="img/5.PNG" height=500>
```
Comment (pattacini) [inline docs/upgrade_kits/realsense_holder/support.md:117]:
```suggestion
<center> <img src="img/5.PNG" height=500> </center>
```

File: docs/upgrade_kits/realsense_holder/support.md [hunk 9]
```diff
@@ -47,45 +54,71 @@ This upgrade kit is meant to improve iCub's 3D perception and grasping capabilit
...
 
 ## Historical review
 This upgrade kit has been developed together with the [HSP@IIT](https://www.iit.it/web/humanoid-sensing-and-perception) research line as an optional plug-in support to improve iCub perception and facilitate it in grasping tasks iCub. For reference, see also the page [design holder realsense](https://github.com/icub-tech-iit/design-holder-realsense).
 
-<center> ![](img/1.PNG) </center>
+<center> <img src="img/1.PNG" height=400> </center>
 
-The holder has been designed to be compatible with the two following RealSense models:
+n designed
+The holder has bee to be compatible with the following RealSense models:
```
Comment (pattacini) [inline docs/upgrade_kits/realsense_holder/support.md:73]:
```suggestion
The holder has been designed to be compatible with the following RealSense models:
```

File: docs/upgrade_kits/realsense_holder/support.md (hunk 8 above)
Comment (pattacini) [reply docs/upgrade_kits/realsense_holder/support.md:117]: You could commit the suggestion straight away @Mick3Lozzo.

**Review by pattacini — APPROVED** [review]
Fine with me so far from a rendering standpoint, @Miche19 👍🏻 
At any rate, we should wait for high-level reviews on the content.

**Review by pattacini — CHANGES_REQUESTED** [review]
Hi @Mick3Lozzo 

Could you please replace the files archived in this zip?
[amends.zip](https://github.com/icub-tech-iit/documentation/files/11476844/amends.zip)

The images are such that they're shown equally well in the first table. 
Also, please, remove the unused file `7.PNG`.

File: docs/upgrade_kits/realsense_holder/support.md [hunk 10]
```diff
@@ -3,10 +3,14 @@
...
-<center> ![](img/8.PNG) </center>
+It ensures the compatibility with more RealSense models.
 
+|       |       |      |
+| :---: | :---: | :---: |
+| <img src="../img/11.PNG"> | <img src="../img/8.PNG"> | <img src="../img/9.PNG"> |
+|                 |                |         |
 |       |       	          |
 |   :--- |    :-----------           |
 |    IIT alias (used as ordering reference)| KIT_008  |
```
Comment (pattacini) [inline docs/upgrade_kits/realsense_holder/support.md:16]:
```suggestion
|  |  |  |
| :---: | :---: | :---: |
| <img src="../img/11.PNG"> | <img src="../img/8.PNG"> | <img src="../img/9.PNG"> |

|  |  |
| :--- | :--- |
|  IIT alias (used as ordering reference) | KIT_008  |
```

**Review by pattacini — APPROVED** [review]

**Pull request merged**